src/qml/doc/translator-comments.md

```diff
@@ -0,0 +1,86 @@
+# Translator Comment Policy
+
+Translator comments are required for all user-facing strings. A PR introducing
+new user-facing strings must also submit the accompanying translator comment.
+To prevent this requirement from halting development, the PR author can specify
+that they would like the introduction of the comment to be picked up in a
+follow-up PR.
```

Comment:
I understand that this might be part of official guidelines and must be mentioned here. But I was just curious, wouldn’t this guideline indirectly motivate a contributor to laze out on translator comments and put it for future PR, potentially increasing translation burden on the complete project.